src/main/java/Duke.java

```diff
@@ -1,10 +1,169 @@
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+import java.io.File;
```

Comment:
Perhaps you might want to reorder them due to a possible lexicographical ordering issue?